python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vxlan_interface.schema.yml

```diff
@@ -84,17 +84,20 @@ keys:
                   type: dict
                   keys:
                     id:
-                      type: int
+                      type: str
```

Comment:
let's add a new key `vlan_ranges` at the same level as `vlans` to handle the ranges.
New model would only take a range from vni

Reply:
Added below model:
```
vlan_range:
  vlans: str
  vnis: str
  ```